coap-endpoint/src/main.rs

```diff
@@ -0,0 +1,225 @@
+mod auth;
+mod command;
+mod downstream;
+mod error;
+mod response;
+mod telemetry;
+
+//mod command;
+//mod server;
+
+use crate::auth::DeviceAuthenticator;
+use crate::error::CoapEndpointError;
+use crate::response::Responder;
+use dotenv::dotenv;
+use drogue_cloud_endpoint_common::{
+    command_endpoint::{CommandServer, CommandServerConfig},
+    commands::Commands,
+    downstream::{DownstreamSender, DownstreamSink, KafkaSink},
+    error::EndpointError,
+};
+use drogue_cloud_service_common::{
+    config::ConfigFromEnv,
+    health::{HealthServer, HealthServerConfig},
+};
+use futures::{self, TryFutureExt};
+use std::collections::LinkedList;
+use std::ops::DerefMut;
+use telemetry::PublishOptions;
+//use http::HeaderValue;
+//use log;
+use std::net::SocketAddr;
+
+//use bytes::Bytes;
+//use bytestring::ByteString;
+use coap::Server;
+use coap_lite::{CoapOption, CoapRequest, CoapResponse};
+use serde::Deserialize;
+
+#[derive(Clone, Debug, Deserialize)]
+pub struct Config {
+    #[serde(default)]
+    pub disable_tls: bool,
+    #[serde(default)]
+    pub cert_bundle_file: Option<String>,
+    #[serde(default)]
+    pub key_file: Option<String>,
```

Comment:
Remove unused options

Reply:
so I'm aware DTLS is still ways off, but I've left these in for that purpose. I can still remove them if you want

Reply:
Yes, please remove it, as it can easily be added back later.